### PR TITLE
Clean up old test:ci* script

### LINKED ISSUE #2152: npm run test:full is not supported on Windows
`npm run test:full` is not supported on Windows, I get this output:

```
>npm run test:full

> root@0.14.0 test:full C:\src\mf\FluidFramework
> npm run test:ci:coverage


> root@0.14.0 test:ci:coverage C:\src\mf\FluidFramework
> npm run test:coverage;t1=$?;npm run test:copyresults; exit $t1

npm ERR! missing script: test:coverage;t1=$?;npm
npm ERR!
npm ERR! Did you mean this?
npm ERR!     test:coverage

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\mfields\AppData\Roaming\npm-cache\_logs\2020-05-12T21_56_44_786Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! root@0.14.0 test:ci:coverage: `npm run test:coverage;t1=$?;npm run test:copyresults; exit $t1`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the root@0.14.0 test:ci:coverage script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\mfields\AppData\Roaming\npm-cache\_logs\2020-05-12T21_56_44_826Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! root@0.14.0 test:full: `npm run test:ci:coverage`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the root@0.14.0 test:full script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\mfields\AppData\Roaming\npm-cache\_logs\2020-05-12T21_56_44_873Z-debug.log
```

### PR DESCRIPTION
**Pull request by curtisman**
CI npm script should have ci:*.   
Since our CI are running on linux, updated instruction on how to replicate it and give different instruction for windows in the README.md.  (and note that the instruction is just different for windows, not that it doesn't support windows)

Remove test:full. Close #2152

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,12 +106,13 @@ npm run test:coverage
 
 ### Mimic the official CI build
 
-_Note: The official build uses npm run test:full, which doesn’t work on Windows._
+Our CI pipelines runs on linux machines, and the npm scripts all have the `ci` prefix.
```
Comment (tylerbutler) [inline README.md:109]:
```suggestion
Our CI pipelines run on Linux machines, and the npm scripts all have the `ci` prefix.
```

File: README.md [hunk 2]
```diff
@@ -106,12 +106,13 @@ npm run test:coverage
...
+To replicate the test steps from the CI pipeline locally, run the following commands for the packages or lerna repos:
```
Comment (tylerbutler) [inline README.md:110]:
```suggestion
To replicate the test steps from the CI pipeline locally, run the following commands for the packages or Lerna monorepos:
```

**Review by tylerbutler — APPROVED** [review]
:shipit:

**Review by markfields — APPROVED** [review]
🎂 yayyyy.  I was thinking about deleting all the `test:ci` and similar scripts too as I familiarized myself with this last week working on my PR.

**Pull request merged**